stan/math/rev/scal/fun/round.hpp

```diff
@@ -16,7 +16,7 @@ namespace stan {
           op_v_vari(::round(avi->val_), avi) {
         }
         void chain() {
-          if (unlikely(boost::math::isnan(avi_->val_)))
+          if (unlikely(is_nan(avi_->val_)))
```

Comment:
No, it won't wreak havoc.

> On Sep 1, 2016, at 2:44 AM, Daniel Lee <notifications@github.com> wrote:
> 
> In stan/math/rev/scal/fun/round.hpp:
> 
> > @@ -16,7 +16,7 @@ namespace stan {
> >            op_v_vari(::round(avi->val_), avi) {
> >          }
> >          void chain() {
> > -          if (unlikely(boost::math::isnan(avi_->val_)))
> > +          if (unlikely(is_nan(avi_->val_)))
> 
> ok. Is this pull request going to wreak havoc on your other branch? If it is, I can kill this one. It was only an hour or two of work. Or... I can help with the merge.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>